src/Icon.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component, useState } from "react";
+import { View, Text, Platform } from "react-native";
+import AndroidIcon from 'react-native-vector-icons/MaterialCommunityIcons';
+import { SFSymbol } from "./SFSymbols";
+
+// For Icons, we use SF Symbols on iOS and Google's Material Design Icons on
+// Android. The IconNames variable below takes a generic name and maps to the
+// name of the icon on each platform.
+//
+// To find SF Symbols, you can use the SF Symbols browser on the App Store.
+// To find Material icons, use the browser here: https://materialdesignicons.com
+
+const IconNames = {
+  "publish": { ios: "square.and.pencil", android: "notebook-edit-outline" },
```

Comment:
At this small size, I don't think `notebook-edit` reads very well, so I'm going to try just using `pencil` here on Android.